packages/interface/src/features/results/components/ResultItem.tsx

```diff
@@ -39,7 +49,11 @@ export const ResultItem = ({ pollId, rank, project, medal = undefined }: IResult
 
         <div className="mx-2 flex-1">{metadata.data?.name}</div>
 
-        <div className="w-24 flex-none text-end">{`${project.votes} votes`}</div>
+        {totalVotes ? (
+          <div className="w-24 flex-none text-end">{`${((budget * project.votes) / totalVotes).toFixed(2)} $`}</div>
```

Comment:
then do we need to specify the unit here? like `250 USD$`, or I would be confused whether we use the real world fiat or any coins

Reply:
This is the last question I have, if this done, could approve it!

Reply:
I added the dollar sign, but can add usd too

Reply:
I saw it, maybe just me not familiar with using USD currency, seems reasonable for me